web/src/components/layout.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Header from './header';
+
+export default function Layout({ children }) {
+    return (
+        <>
+            <div>
+                <Header />
+                {children}
+            </div>
+        </>
```

Comment:
I don't think we need to wrap this in a fragment and a `<div>`.  I think I usually do something like 
```
<>
    <Header>
    <main>{children}</main>
</>
```